state/application.go

```diff
@@ -272,10 +272,6 @@ func (a *Application) removeOps(asserts bson.D) ([]txn.Op, error) {
 			Id:     a.doc.DocID,
 			Assert: asserts,
 			Remove: true,
-		}, {
-			C:      settingsC,
-			Id:     a.settingsKey(),
-			Remove: true,
```

Comment:
from the pastebin I see 3 calls to remove the settings key. This seems to be removing one of them. Do you know where the other is?
Are we sure this is the only path that triggers this code?